basyx/aas/adapter/http.py

```diff
@@ -355,6 +355,15 @@ def request_body(cls, request: Request, expect_type: Type[T], stripped: bool) ->
             return cls.json(request.get_data(), expect_type, stripped)
         return cls.xml(request.get_data(), expect_type, stripped)
 
+    @staticmethod
+    def decode_specific_asset_ids(encoded_ids: str):
+        # Decoding from base64url and transforming to JSON
+        decoded_json = base64.urlsafe_b64decode(encoded_ids + '===').decode('utf-8')
+        # Assuming the JSON represents a list of SpecificAssetId objects
+        specific_asset_ids = json.loads(decoded_json)
+        # Transform the list of dictionaries into SpecificAssetId instances
+        return [model.SpecificAssetId(**id_dict) for id_dict in specific_asset_ids]
+
```

Comment:
This method doesn't work (correctly) and should be removed. Instead, the function `base64urljson_list`, implemented in #22, should be used.